VRVU-122:Add button to take a registration photo

sherrif10 opened the pull request:
PR https://sd-vxnaid.atlassian.net/browse/VRVU-122  cc @pwargulak  @jdolkowski  @jdolkowski

sherrif10 committed:
VRVU-122:Add button to take a registration photo

sherrif10 committed:
more clear

sherrif10 committed:
change layout

jdolkowski commented:
Code looks good. However try to make UI look better once the photo is there. Button takes too much space. Maybe move it under the photo and make it smaller? Be creative and good luck!

sherrif10 commented:
> Code looks good. However try to make UI look better once the photo is there. Button takes too much space. Maybe move it under the photo and make it smaller? Be creative and good luck!

Hello @jdolkowski  , Addressed your comment..

sherrif10 committed:
add camera icon
